src/org/usfirst/frc4904/robot/subsystems/BallIO.java

```diff
@@ -0,0 +1,46 @@
+package org.usfirst.frc4904.robot.subsystems;
+
+
+import org.usfirst.frc4904.standard.commands.Idle;
+import org.usfirst.frc4904.standard.subsystems.motor.Motor;
+import org.usfirst.frc4904.standard.subsystems.motor.ServoSubsystem;
+import edu.wpi.first.wpilibj.command.Subsystem;
+
+public class BallIO extends Subsystem {
+	public final Motor directionalRoller;
```

Comment:
This and the below four constants should not be public. At best, they should be protected.